plugins/rollbar/app/models/rollbar_notification.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'faraday'
+
+class RollbarNotification
+  def initialize(webhook_url:, access_token:, environment:, revision:)
+    @webhook_url = webhook_url
+    @access_token = access_token
+    @environment = environment
+    @revision = revision
+  end
+
+  def self.deliver(webhook_url:, access_token:, environment:, revision:)
```

Comment:
can we handle delivery right here instead of 8+ lines of extra passing things around to the instance ?